microsoft/PowerToys
# [Dev files Preview] Handle access denied error and properly show erro…

…r messages


## Summary of the Pull Request


## PR Checklist

- [x] **Closes:** #23957


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed

 - Take a file with an extension supported by Dev files preview
 - go to properties -> security tab-> set Full control permission to Deny
 - Try previewing the file
 - observe that there is no crash and proper error message is shown
